src/reactviews/pages/ConnectionDialog/azureBrowsePage.tsx

```diff
@@ -1,9 +1,334 @@
+                                setSelectedDatabase(db);
+                                setConnectionProperty("database", db ?? "");
+                            },
+                            currentValue: selectedDatabase,
+                        }}
+                    />
+                    {context.state.connectionComponents.mainOptions
+                        .filter((opt) => !["server", "database"].includes(opt))
+                        .map((inputName, idx) => {
+                            const component =
+                                context.state.connectionComponents.components[
+                                    inputName as keyof IConnectionDialogProfile
+                                ];
+                            if (component.hidden === true) {
+                                return undefined;
+                            }
+
+                            return (
+                                <FormField
+                                    key={idx}
+                                    context={context}
+                                    component={
+                                        component as FormItemSpec<IConnectionDialogProfile>
+                                    }
+                                    idx={idx}
+                                    props={{ orientation: "horizontal" }}
+                                />
+                            );
+                        })}
+                </>
+            )}
+
+            <AdvancedOptionsDrawer
+                isAdvancedDrawerOpen={isAdvancedDrawerOpen}
+                setIsAdvancedDrawerOpen={setIsAdvancedDrawerOpen}
+            />
+            <div className={formStyles.formNavTray}>
+                <Button
+                    shape="square"
+                    onClick={(_event) => {
+                        setIsAdvancedDrawerOpen(!isAdvancedDrawerOpen);
+                    }}
+                    className={formStyles.formNavTrayButton}
+                >
+                    {locConstants.connectionDialog.advancedSettings}
+                </Button>
+                <div className={formStyles.formNavTrayRight}>
+                    <TestConnectionButton
+                        className={formStyles.formNavTrayButton}
+                    />
+                    <ConnectButton className={formStyles.formNavTrayButton} />
+                </div>
+            </div>
+        </div>
+    );
+};
+
+const AzureBrowseDropdown = ({
+    label,
+    required,
+    clearable,
+    content,
+}: {
+    label: string;
+    required?: boolean;
+    clearable?: boolean;
+    content: {
+        valueList: string[];
+        setValue: (value: string | undefined) => void;
+        currentValue?: string;
+    };
+}) => {
+    const formStyles = useFormStyles();
+
+    return (
+        <div className={formStyles.formComponentDiv}>
+            <Field label={label} orientation="horizontal" required={required}>
+                <Dropdown

```

Comment:
Can we switch this to Combobox for auto complete?  I am guessing the list can be quite long for most people. 

https://react.fluentui.dev/?path=/docs/components-combobox--docs

Reply:
Aha, good call.  I didn't realize this was a separate control, and was going to look into textbox filtering.

[This](https://react.fluentui.dev/?path=/docs/components-combobox--docs#filtering) seems like exactly what we want.